src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBOptions.java

```diff
@@ -35,12 +36,16 @@ public GenomicsDBOptions(final Path reference, final GenomicsDBArgumentCollectio
         this.useBCFCodec = genomicsdbArgs.useBCFCodec;
         this.sharedPosixFSOptimizations = genomicsdbArgs.sharedPosixFSOptimizations;
         this.useGcsHdfsConnector = genomicsdbArgs.useGcsHdfsConnector;
+        if (genomicsdbArgs.maxDiploidAltAllelesThatCanBeGenotyped - 1 < genotypeCalcArgs.maxAlternateAlleles) {
```

Comment:
Include a comment explaining the rationale behind the "must be at least one greater than the genotype calculation max alt alleles" guideline. I assume it's just adjusting for NON_REF?